infra/update.go

```diff
@@ -0,0 +1,44 @@
+package infra
+
+import (
+	"github.com/gravitational/robotest/lib/defaults"
+	"github.com/gravitational/robotest/lib/wait"
+	"github.com/gravitational/trace"
+	"golang.org/x/crypto/ssh"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+// UploadUpdate started uploading new application update
```

Comment:
s/started/starts
`UploadUpdate uploads a new application version to the installer node`